registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java

```diff
@@ -662,9 +658,16 @@ private void validateBiometricRecord(BiometricRecord biometricRecord, List<Strin
 		boolean isBioFound = false;
 		for (String biometricSegment : biometricModalitySegmentsMap.keySet()) {

```

Comment:
Lets rename biometricSegment to biometricModality

Reply:
fixed